fabric/src/main/java/org/valkyrienskies/mod/fabric/mixin/feature/explosions/ClipContextMixin.java

```diff
@@ -0,0 +1,29 @@
+package org.valkyrienskies.mod.fabric.mixin.feature.explosions;
+
+import net.minecraft.world.entity.Entity;
+import net.minecraft.world.level.ClipContext;
+import net.minecraft.world.phys.shapes.CollisionContext;
+import org.spongepowered.asm.mixin.Final;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Redirect;
+
+@Mixin(ClipContext.class)
+public class ClipContextMixin {
```

Comment:
Can we just edit your code to not pass in null values instead of this mixin?

Reply:
Are you sure there's no other clip method you can use that doesn't require an entity?

Reply:
As far as I can tell, there's no version of clip that doesn't use a ClipContext, and ClipContext requires an entity. All this mixin does is make parity with the forge implementation of ClipContext.

Reply:
If there's truly no way to do this without a mixin at least add a comment explaining that